cki-send.py

```diff
@@ -40,36 +40,54 @@ def main(args):
             for line in report_file.readlines():
                 test, status, _ = line.split()
 
-                console_url = f"{args.build_url}artifact/output/{instance_type}-{test}.console"
-                result_file_name = f'{instance_type}-{test}.result'
-                result_url = f"{args.build_url}artifact/results/{result_file_name}"
-
-                msg["tests"].append({
+                common_data = {
                     "build_id": args.kcidb_build_id,
-                    "id": f'redhat:virt_qe_s1.{args.build_id}.{instance_type}.{test}',
                     "origin": "redhat",
                     "environment": {
                         "comment": f'Instance {instance_type} of {args.cloud}'
                     },
-                    "path": f"ltp.{test}",
-                    "log_url": console_url,
-                    "status": status,
                     "waived": True,
-                    "output_files": [
-                        {
-                            "name": "web_gui",
-                            "url": f"{args.build_url}display/redirect",
-                        },
-                        {
-                            "name": "consoleText",
-                            "url": f"{args.build_url}consoleText",
-                        },
-                        {
-                            "name": result_file_name,
-                            "url": result_url,
-                        },
-                    ],
-                })
+                }
+
+                common_output_files = [
+                    {
+                        "name": "web_gui",
+                        "url": f"{args.build_url}display/redirect",
+                    },
+                    {
+                        "name": "consoleText",
+                        "url": f"{args.build_url}consoleText",
+                    },
+                ]
+
+                if test == "LTP_test":
```

Comment:
```suggestion
                if test == "LTP_test" and status == "ABORT":
                    # When LTP_test is reported, means that the tests failed and we won't get subtests.
```
Or something like that, to make it simpler to understand, otherwise seems like "if it's LTP it's error".

Reply:
Also added a comment about ABORT->ERROR